Build failure

[ERROR] fireTorpedo_All_Success  Time elapsed: 0.013 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at hu.bme.mit.spaceship.GT4500Test.fireTorpedo_All_Success(GT4500Test.java:37)


fireTorpedo case ALL not implemented

--- a/src/main/java/hu/bme/mit/spaceship/GT4500.java
+++ b/src/main/java/hu/bme/mit/spaceship/GT4500.java
@@ -78,7 +78,9 @@ public class GT4500 implements SpaceShip {
 
       case ALL:
         // try to fire both of the torpedo stores
-        //TODO implement feature
+        if ((!primaryTorpedoStore.isEmpty()) && (!secondaryTorpedoStore.isEmpty())) {
+          firingSuccess = ((primaryTorpedoStore.fire(1) && secondaryTorpedoStore.fire(1)));
+        }
 
         break;
     }

